src/atopile/cli/build.py

```diff
@@ -68,14 +95,97 @@ def do_prebuild(build_ctx: BuildContext) -> None:
     with ExceptionAccumulator() as accumulator:
         # Solve the unknown variables
         if not build_ctx.dont_solve_equations:
-            with accumulator.collect():
-                atopile.assertions.simplify_expressions(build_ctx.entry)
-                atopile.assertions.solve_assertions(build_ctx)
-                atopile.assertions.simplify_expressions(build_ctx.entry)
+            # with accumulator.collect():
+            #     atopile.assertions.simplify_expressions(build_ctx.entry)
+            #     atopile.assertions.solve_assertions(build_ctx)
+            #     atopile.assertions.simplify_expressions(build_ctx.entry)
+            raise errors.AtoNotImplementedError(
+                "Equation solving is not implemented yet"
+            )
+
+
+def import_from_path(module_name, file_path):
+    # setting to a sequence (and not None) indicates that the module is a package, which lets us use relative imports for submodules
+    submodule_search_locations = []
+
+    spec = importlib.util.spec_from_file_location(
+        module_name, file_path, submodule_search_locations=submodule_search_locations
+    )
+    if spec is None:
+        raise RuntimeError(f"Failed to load module {module_name} from {file_path}")
```

Comment:
FileNotFound error? Or `errors.AtoFileNotFoundError` (off the cuff, may not exist)

Then it's present nicely to the user